Ensure that all Qt headers are reached during the build process.

Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>

--- QScintilla_gpl-2.10.3/Python/configure.py.orig_	2018-03-19 17:27:18.661667989 -0300
+++ QScintilla_gpl-2.10.3/Python/configure.py	2018-03-19 17:32:18.845673621 -0300
@@ -1545,6 +1545,10 @@ INSTALLS += sip
     if includepath:
         pro.write('INCLUDEPATH += %s\n' % includepath)
 
+    for subdir in glob.glob("/usr/include/qt/*"):
+        if os.path.isdir(subdir) or os.path.islink(subdir):
+            pro.write('INCLUDEPATH += {}\n'.format(subdir))
+
     # Make sure the SIP include directory is searched before the Python include
     # directory if they are different.
     pro.write('INCLUDEPATH += %s\n' % quote(target_config.sip_inc_dir))
